index.js

```diff
@@ -55,15 +55,30 @@ app.post('/', upload.single('thumb'), async(req, res, next) => {
   if (payload.event === 'media.play' || payload.event === 'media.rate') {
     if (image) {
       console.log('[REDIS]', `Using cached image ${key}`);
-    } else if (!image && req.file && req.file.buffer) {
-      console.log('[REDIS]', `Saving new image ${key}`);
-      image = await sharp(req.file.buffer)
-        .resize(75, 75)
-        .background('white')
-        .embed()
-        .toBuffer();
-
-      redis.set(key, image, 'EX', SEVEN_DAYS);
+    } else {
+      let buffer;
+      if (req.file && req.file.buffer) {
+        buffer = req.file.buffer;
+      } else if (payload.thumb) {
+        console.log('[REDIS]', `Retrieving image from  ${payload.thumb}`);
+        buffer = await request.get({
+          uri: payload.thumb,
+          encoding: null
+        });
+      }
+      if (buffer) {
+        image = await sharp(buffer)
+          .resize({
```

Comment:
This change to the `sharp` processing is due to deprecation warnings of the old chaining of `background` and `embed`. 

https://github.com/lovell/sharp/issues/1392